lib/krane/kubernetes_resource/horizontal_pod_autoscaler.rb

```diff
@@ -16,7 +16,7 @@ def deploy_failed?
     end
 
     def kubectl_resource_type
-      'hpa.v2beta1.autoscaling'
+      'hpa.v2.autoscaling'
```

Comment:
I wonder if we can use `horizontalpodautoscalers.autoscaling` and avoid issues with the api version used.